scripts/Diffraction/isis_powder/routines/common.py

```diff
@@ -119,6 +126,36 @@ def generate_run_numbers(run_number_string):
     return run_list
 
 
+def generate_splined_name(vanadium_string, *args):
+    """
+    Generates a unique splined vanadium name which encapsulates
+    any properties passed into this method so that the vanadium
+    can be later loaded.
+    :param vanadium_string: The name of this vanadium run
+    :param args: Any identifying properties to append to the name
+    :return: The splined vanadium name
+    """
+    out_name = "VanSplined" + '_' + str(vanadium_string)
+    for value in args:
+        out_name += '_' + str(value)
+
+    out_name += ".nxs"
+    return out_name
```

Comment:
Typically 40-60 characters at the most. These are used internally for the script. Basically we generate a unique fingerprint for a splined vanadium containing the offset name, tt_mode (on Pearl) and vanadium numbers. 
This also means users can see something of the format
`VanSplined_12345-12350_offsets_16_5.cal.nxs`